src/matlab.ts

```diff
@@ -65,9 +66,31 @@ export async function runCommand(hs: HelperScript, platform: string, fn: ExecFn)
  * Get the path of the script containing RunMATLABCommand for the host OS.
  *
  * @param platform Operating system of the runner (e.g., "win32" or "linux")
- */
-export function getRunMATLABCommandScriptPath(platform: string): string {
-    const ext = platform === "win32" ? "bat" : "sh";
-    const rmcPath = path.join(__dirname, "bin", `run_matlab_command.${ext}`);
+ * @param architecture Architecture of the runner (e.g., "x64")
+*/
+export function getRunMATLABCommandScriptPath(platform: string, architecture: string): string {
+    if (architecture != "x64") {
+        throw new Error(`This action is not supported for runners of platform ${platform} and architecture ${architecture}.`);
```

Comment:
@mw-hrastega any thoughts on this error message?

Reply:
Could you give me an example of the message holes? Will each hole be filled with a scalar or a list?

If both are scalar, maybe something like this?

This action is not supported on ${platform} runners using the ${architecture} architecture.

Reply:
I think that sounds reasonable. Yes, they will both be scalar. One example of such an error would read `This action is not supported on windows runners using the x86 architecture.`

Reply:
Sounds good to me.